src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import "./Navbar.scss"
+
+
+function Navbar() {
+  return (
+    <div className="Navbar">
+         <div className="navbar__left">
+            <div className="navbar__icon"><img alt="Logo" src="assets/skillChain.svg"/></div>
+        </div>
+        <div className="navbar__right">
+            <div className="navbar__links"><a href="#">Find Candidates</a></div>
+            <div className="navbar__links"><a href="#">Find Jobs</a></div>
+            <div className="navbar__links"><a href="#">Contact Us</a></div>
```

Comment:
there is no 'Contact Us' in the design))

Reply:
There is, but I think it moved out of the main picture

![Screenshot (220)](https://user-images.githubusercontent.com/121715056/220582487-3d7f45c3-af9e-48ff-acd0-8bb2bce4132d.png)

Reply:
oh, sorry for confusion then) ignore my prev message